src/Frontend/Page/Idea.hs

```diff
@@ -181,20 +177,10 @@ instance FormPage CreateIdea where
                             div_ [class_ "category-radios"] $ do
                                 DF.inputRadio True "idea-category" v
                             div_ [class_ "icon-list m-inline category-image-select"] $ do
-                                ul_ $ do
+                                ul_ $ toHtml `mapM_` [(minBound :: Category)..]
```

Comment:
Please introduce a newtype for this, converting a category value to special list item only for this module, in my opinion has to be in a newtype.